clj-refactor.el

```diff
@@ -2199,6 +2200,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-sans-ns-keyword-p ()
```

Comment:
I really dislike this name.  I think a better name would be `cljr--in-keyword-sans-alias?`.  We're using `?` for predicates everywhere else, so we should be consistent.